jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -954,6 +1035,17 @@ void endpointExpired(AbstractEndpoint endpoint)
         }
     }
 
+    /**
+     * Notifies this conference that one of its relays has expired.
+     *
+     * @param relay the <tt>Relay</tt> which expired.
+     */
+    public void relayExpired(Relay relay)
+    {
+        String id = relay.getId();
```

Comment:
I think we need to call `removeReceiver` for all local endpoints here (like in `endpointExpired`).

Reply:
`Conference.endpointExpire` does call `removeReceiver`, unless I"m missing something?

Reply:
Oh, right, the receiver in this case is the relay itself, not the relayed endpoint.  Added.

Reply:
Oh, right, the receiver in this case is the relay itself, not the relayed endpoint.  Added.